style.css

```diff
@@ -1,825 +1,811 @@
+    font-size: 32px;
+    line-height: 1;
+    padding-top: 10px;
+    margin-bottom: 13px;
   }
 
-  .header__menu {
-    background: #ededed;
-    z-index: 3;
-    height: 140px;
-    width: 140px;
+  .second__container {
+    width: 306px;
+    height: 348px;
   }
 
-  .header__menu-item {
-    height: 4px;
-    width: 48.7px;
-    margin-bottom: 10px;
+  .second__subtitle {
+    font-size: 13px;
+    margin-bottom: 46px;
   }
 
-  .header__logo p {
-    margin: 25px 21px;
-    color: black;
-    font-size: 60px;
-    letter-spacing: 4px;
+  .second__text {
+    font-size: 12px;
+    margin-bottom: 32px;
   }
 
-  .text__title {
-    position: inherit;
-    align-items: flex-end;
-    font-size: 98px;
-    line-height: 103px;
-    padding-left: 52px;
-    letter-spacing: 5px;
+  .second__block-color-gray,
+  .second__block-color-pink,
+  .second__block-color-blue,
+  .second__block-color-green {
+    width: 24px;
+    height: 24px;
   }
 
-  .text__container {
-    display: flex;
-    align-items: flex-end;
-    width: 100%;
-    max-width: 750px;
-    height: 945px;
-    letter-spacing: 5px;
+  .second__block-color-pink {
+    margin-left: 11.82px;
   }
 
-  .step {
-    display: flex;
+  .second__block-color-blue {
+    margin-left: 11.82px;
   }
 
-  .text {
-    width: 100%;
-    max-width: 750px;
-    height: 600px;
+  .second__block-color-green {
+    margin-left: 11.82px;
   }
 
-  .vectormob {
-    position: absolute;
-    width: 67px;
-    height: 39px;
-    left: 345px;
-    top: 1016px;
-    display: block;
+  .second__block-coust {
+    font-size: 34px;
+    margin-right: 20px;
   }
-}
 
-@media (max-width: 660px) {
-  .first-container {
-    padding: 267px 0;
-    background: url(img/pete-bellis-432867-unsplashmob.png) no-repeat;
-    background-size: 660px;
-    background-position: center;
+  .second__quantity {
+    text-decoration: none;
+    color: #000;
+    display: block;
+    font-size: 15px;
+    border: 1.5px solid #000;
+    padding: 7px 25px;
+    margin-right: 9px;
+    margin-left: 44px;
   }
 
-  .header__inner {
-    width: 100%;
-    height: 123px;
-    background: #fff;
-    z-index: 2;
+  .second__add {
+    margin-left: 0;
+    border: 1.5px solid #000;
+    padding: 11px 45px;
+    font-size: 10px;
   }
 
-  .header__menu {
-    background: #ededed;
-    z-index: 3;
-    height: 123px;
-    width: 123px;
+  .second__btn {
+    display: flex;
   }
 
-  .header__menu-item {
-    height: 4px;
-    width: 47.7px;
-    margin-bottom: 7px;
+  .second__other {
+    font-weight: 500;
+    font-size: 13px;
+    line-height: 1.2;
+    text-decoration: none;
+    color: #000;
+    margin-left: 5px;
+    margin-right: 10px;
+    margin-top: 102px;
+    display: inline-block;
   }
 
-  .header__logo p {
-    margin: 22px 16px;
-    color: black;
-    font-size: 53px;
-    letter-spacing: 4px;
+  .second__vector {
+    display: inline-block;
   }
-
-  .text__container {
-    display: flex;
-    align-items: flex-end;
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)